acceptance/auth/bundle_and_profile/script

```diff
@@ -0,0 +1,26 @@
+# Replace placeholder with an actual host URL
+envsubst < databricks.yml > out.yml && mv out.yml databricks.yml
+envsubst < .databrickscfg > out && mv out .databrickscfg
+
+title "Inside the bundle, no flags"
+DATABRICKS_CONFIG_FILE=.databrickscfg trace errcode $CLI current-user me | jq .userName
```

Comment:
nit: can exported once, since it's the same for every command:

`export DATABRICKS_CONFIG_FILE=.databrickscfg`

Reply:
@denik is the environment variables are cleaned up after the tests? Or should I unset the variables manually?

Reply:
No need to unset, each test gets its own shell.